### PR TITLE
Change the bene features hub page to have installation buttons etc

### PR DESCRIPTION
**Pull request by unclegcb**
@mariacha or @mortenson any interest in having a look at this? The existence of the sample module makes this page a little easier to try out if you want to do that. Probably easiest to test by also grabbing [`bene_features` branch](https://github.com/thinkshout/bene_promo_redirect/tree/bene_features) of `bene_promo_redirect`.

The install/uninstall behavior here is a little freewheeling, perhaps, but it is locked out of impacting any module outside the `bene_` namespace.

### CODE REVIEW

File: modules/bene_features_example/bene_features_example.module [hunk 1]
```diff
@@ -19,6 +19,13 @@ function bene_features_example_help($route_name, RouteMatchInterface $route_matc
         'details' => [
           '#markup' => t('Example help page for Bene Features Example.'),
         ],
+        'video' => [
+          '#type' => 'inline_template',
+          '#template' => '{{ helpvideo | raw }}',
+          '#context' => [
+            'helpvideo' => '<div><iframe width="441" height="331" src="https://www.youtube.com/embed/Otbml6WIQPo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>',
```
Comment (mortenson) [inline modules/bene_features_example/bene_features_example.module]: You can also throw all this markup right in `#template` and it should work. I use this trick too for avoiding filtering.

File: src/Controller/BeneFeaturesController.php [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+ */
+class BeneFeaturesController extends ControllerBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function configPage() {
+    $modules = system_rebuild_module_data();
+    $uninstall = \Drupal::request()->query->get('remove') ?: FALSE;
+    if ($uninstall) {
```
Comment (mortenson) [inline src/Controller/BeneFeaturesController.php]: We shouldn't use GET parameters here, they would leave this page open to CSRF. A malicious user could trick an admin into clicking a link or viewing an image that pointed to `/admin/config/bene_features?remove=system`.

Comment (unclegcb) [reply src/Controller/BeneFeaturesController.php]: Great feedback! Fixed it up in a16057e

File: modules/bene_features_example/bene_features_example.module (hunk 1 above)
Comment (unclegcb) [reply modules/bene_features_example/bene_features_example.module]: Is it weird that that feels to me like cheating, but what I did only feels like bending the rules? (I gather they are functionally equivalent.)

Comment (mortenson) [reply modules/bene_features_example/bene_features_example.module]: It's all cheating 😄 The alternative would be to use `#markup` and `#allowed_tags` to allow the iframe tag, I think.

Comment (unclegcb) [reply modules/bene_features_example/bene_features_example.module]: Yeah, that might feel the most honest.

**Pull request merged**